### ISSUE TITLE
Functions could be python scripts

### ISSUE THREAD

**Issue by naaando** [Bat41/Spreadsheet#19]
Would be nice to have python scripts loaded as functions, could be another scripting language also.

**Comment by Bat41** [comment]
Yeah, that's a great idea! I'll probably use Python, Ruby or Lua depending on what is the easiest to use with Vala.

**Comment by danrabbit** [comment]
I think Google Sheets uses Javascript right? I wonder if it's worth trying to stay in line with what people might be familiar with from another spreadsheet program

**Comment by ryonakano** [comment]
Honestly I'm not familiar with script languages, but possibly we can use JavaScriptCore if we go with JavaScript? It looks like to have a language binding on Vala:

https://valadoc.org/javascriptcoregtk-6.0/JSC.Context.html
